api/v1alpha1/ionoscloudmachine_types.go

```diff
@@ -155,6 +155,9 @@ type IonosCloudMachineSpec struct {
 	//+optional
 	AdditionalNetworks Networks `json:"additionalNetworks,omitempty"`
```

Comment:
```suggestion
	// +listType=map
	// +listMapKey=networkID
	AdditionalNetworks Networks `json:"additionalNetworks,omitempty"`
```

Reply:
@farodin91 Please resolve if you are happy with the change